lib/cli/driver.js

```diff
@@ -262,9 +273,88 @@ class DriverCommand {
     return this.config.installedDrivers;
   }
 
-  update (/*{driver}*/) {
-    // TODO implementation
-    return {};
+  async update ({driver, force}) {
+    const all = driver === 'all';
+    // if we're specifically requesting an update for a driver, make sure it's installed
+    if (!all && !this.config.isInstalled(driver)) {
+      throw new Error(`Driver '${driver}' was not installed, so can't be updated`);
+    }
+    const driversToUpdate = all ? Object.keys(this.config.installedDrivers) : [driver];
+    const errors = {};
+    const updates = {};
+    for (const d of driversToUpdate) {
+      try {
+        await spinWith(this.isJsonOutput, `Checking if driver '${d}' is updatable`, () => {
+          if (this.config.installedDrivers[d].installType !== INSTALL_TYPE_NPM) {
+            throw new Error(`d '${d}' was not installed via npm, so we can't check for updates`);
```

Comment:
that is `d`?

Reply:
this error should not be thrown if `all` is set. We should be smart about updatable drivers selection

Reply:
this isn't an 'error' per se, it gets handled and just output as the report for that particular driver.